docs/metrics.asciidoc

```diff
@@ -14,6 +14,12 @@ You can adjust the interval with the setting <<config-metrics-interval>>.
 
 The metrics will be stored in the `apm-*` index and have the `processor.event` property set to `metric`.
 
+NOTE: Dedicated JVM metrics views are available since Elastic stack version 7.1.
+Since 7.5, metrics are aggregated separately for each JVM, relying on the ID of the underlying system- either container ID (where applicable) or hostname.
+Since Java agent version 1.11.0, it is possible to manually configure a unique name for each service node/JVM through
+<<config-service-node-name>>.
```

Comment:
I think the funky parenthesis are occurring because of the "added" tag in the title:
```
==== `service_node_name` (added[1.11.0])
```

You may need to change this line to manually specify the link text:
```suggestion
<<config-service-node-name,`service_node_name`>>.
```